packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -67,7 +65,7 @@ export const DownloadCartPage: React.FunctionComponent<
     refetch()
   }
   return (
-    <div className="DownloadCartPage bootstrap-4-backport">
```

Comment:
This `bootstrap-4-backport` was messing up the new card styling, so narrow the scope